docker/Dockerfile

```diff
@@ -36,12 +35,6 @@ ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/openmpi/lib
 
 ENV NCCL_LAUNCH_MODE=GROUP
 
-#install nccl
```

Comment:
I've decided to remove it. The reason has been explained in https://github.com/InternLM/lmdeploy/pull/2240